docs/onboarding.rst

```diff
@@ -307,3 +307,6 @@ Frequently Asked Questions
 
 What is MS/PFS?
   These are smart contracts to be deployed in the future.  See `Monitoring Service <https://raiden-network-specification.readthedocs.io/en/latest/terminology.html#term-monitoring-service>`_ and `Pathfinding Service <https://raiden-network-specification.readthedocs.io/en/latest/terminology.html#term-pathfinding-service>`_ in the smart contract specification.
+
+How are messages are encoded?
+  The `messages specification <https://raiden-network-specification.readthedocs.io/en/latest/messaging.html>`_ should describe the message formats.  On the implementation, `messages file <https://github.com/raiden-network/raiden/blob/master/raiden/messages.py>`_ contains the details.  See ``encode()`` method of ``Message`` class, and ``packed()`` method of each subclass of ``Message``.
```

Comment:
On the implementation  -> On the implementation side the

Reply:
I did something like that.